.github/workflows/build_hyperion.yml

```diff
@@ -42,13 +36,31 @@ jobs:
       uses: actions/upload-artifact@v2
       with:
         name: webos_libs
-        path: ${{github.workspace}}/hyperion-webos_libvtcapture/build/hyperion-webos_libvtcapture
+        path: ${{github.workspace}}/hyperion-webos/build/hyperion-webos
 
     - name: Upload native components artifacts
       uses: actions/upload-artifact@v2
       with:
         name: webos_libs
-        path: ${{github.workspace}}/hyperion-webos/build/hyperion-webos
+        path: ${{github.workspace}}/servicenative/services.json
+
+    - name: Upload native components artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: webos_libs
+        path: ${{github.workspace}}/hyperion-webos/build/libdile_vt_backend.so
```

Comment:
I think you could just use `path: ${{github.workspace}}/hyperion-webos/build/*_backend.so` here instead of having separate steps.

Reply:
Sure - edited it. Also forgot piccapautostart